### DISCUSSION TITLE
How to control IR leds?

### DISCUSSION THREAD

**Discussion by wes58** [themactep/thingino-firmware#210] [Q&A]
I have recently installed thingino on my wyze v3 cameras so I am still playing with them.
One of the things that I can't seem to be able to do from Web UI is control IR leds.
I have cameras installed behind the glass so when IR leds turn on at night I can't see due to reflection of IR from the glass. Since I have outside lights that are controlled by motion sensor I don't need the IR leds.
I tried to use Web UI preview page but it doesn't seem to be doing anything for IR leds.
I disabled day-night script on Web UI but this doesn't change it as well.
What can I do?

**Comment by wes58** [comment]
Half of the work is done. 
I can disable the IR led when I disable day-night script on Web UI. The only thing I needed to do was to reboot the camera after making changes.
The other issue of manually control IR leds from the Preview page is still not working for me.

**Comment by themactep** [comment]
in console: `irled on`, `irled off`.

**Comment by wes58** [comment]
It didn't work on the WebUI preview page. I have tested it with the latest firmware and it works fine now.

**Comment by themactep** [comment]
ah ok, so it was a cached javascript then.
